### PR TITLE
Improve Russian translation

### PR DESCRIPTION
**Pull request by daudix-UFO**

### CODE REVIEW

**Comment by fabrialberio** [comment]
Thank you so much for your contribution! I really appreciate it.
I'm trying to understand your changes as much as possible before merging them, since they are overwriting another translator's work.
Would you mind telling me what your translations of "No apps found" and "Locale" are changing from the previous ones? I would really appreciate it if you could provide some more information about these changes.
Thank you again for your help!

**Comment by fsobolev** [comment]
@fabrialberio If you don't mind I will review changes in Russian so we can find a compromise, because I disagree with some changes.

File: po/ru.po [hunk 1]
```diff
@@ -89,7 +90,7 @@ msgstr "Добавить новый ключ"
 
 #: ../src/ui/file_page_dialogs.ui.h:2 ../src/file_page.py:89
 msgid "Locale"
-msgstr "Локаль"
+msgstr "Регион"
```
Comment (fsobolev) [inline po/ru.po:93]: В контексте кнопок запуска, речь идёт только про язык, а не все региональные настройки. Раз уж избавляемся от жаргонизма, может лучше просто перевести как "Язык"?

File: po/ru.po [hunk 2]
```diff
@@ -101,15 +102,15 @@ msgstr "Добавить"
 
 #: ../src/ui/file_page_dialogs.ui.h:6
 msgid "Unpin this app?"
-msgstr "Удалить эту кнопку?"
+msgstr "Открепить это приложение?"
 
 #: ../src/ui/file_page_dialogs.ui.h:7
 msgid "If you unpin this app, it will be deleted forever."
-msgstr "Кнопка для этого приложения будет удалена навсегда."
+msgstr "Если вы открепите это приложение, оно будет навсегда удалено."
```
Comment (fsobolev) [inline po/ru.po:109]: Здесь "оно" отсылает к "приложение", что некорректно и запутывает.

**Comment by fabrialberio** [comment]
Feel free to do so, both of  you are certainly more qualified than me on this topic.

File: po/ru.po [hunk 3]
```diff
@@ -157,23 +158,23 @@ msgstr "Перезагрузить приложения"
 
 #: ../src/ui/window.ui.h:7
 msgid "Keyboard Shortcuts"
-msgstr "Сочетания клавиш"
+msgstr "Комбинации клавиш"
 
 #: ../src/ui/window.ui.h:8
 msgid "About PinApp"
-msgstr "О приложении"
+msgstr "О PinApp"
```
Comment (fsobolev) [inline po/ru.po:165]: Обычно приложения GNOME переводят такие строки всегда как "О приложении".

**Review by fsobolev — CHANGES_REQUESTED** [review]

**Comment by fsobolev** [comment]
@fabrialberio I didn't give much attention to the string `"If you unpin this app, it will be deleted forever."` when I created my translation, but I assume it should be changed in English. As I understand it, it basically means "if you unpin this app, **this app** will be deleted forever", which is not what the application actually do, because it deletes a pin, not an app.

File: po/ru.po (hunk 1 above)
Comment (daudix-UFO) [reply po/ru.po:93]: Согласен, так будет понятнее

File: po/ru.po (hunk 3 above)
Comment (daudix-UFO) [reply po/ru.po:165]: В GNOME это встречается в обоих вариантах, например консоль и центр приложений используют "О <имя приложения>"

Comment (fsobolev) [reply po/ru.po:166]: Действительно. Я недостаточно приложений осмотрел когда изучал этот вопрос :) И хотя я всё же предпочёл бы свой вариант, раз уж оба корректны, пусть будет так

**Comment by daudix-UFO** [comment]
Nautilus uses the following wording for permanent deletion of an item

image

File: po/ru.po (hunk 3 above)
Comment (daudix-UFO) [reply po/ru.po]: Вернул ваш вариант ;)

**Comment by fabrialberio** [comment]
How about this:
immagine

**Comment by daudix-UFO** [comment]
I think this makes much more sense, I like it!

**Comment by fsobolev** [comment]
I agree, it's good

**Comment by fabrialberio** [comment]
Ok, I'll update the pot then.

**Review by fsobolev — APPROVED** [review]

**Comment by fsobolev** [comment]
@fabrialberio If you really need detailed explanations on every single change, I could provide them to you, but anyway I confirm that these changes improve translation quality, so it's ready to be merged.

@daudix-UFO thank you! I will work on improving my translator skills so no extra work will be required 😅

**Comment by fabrialberio** [comment]
The fact that you two reached an agreement on them is convincing enough for me, I won't ask for an explanation.
Thanks again for your work!

**Pull request merged**